test/functional/system/transactions/2.delegates/3.different.accounts.same.username.js

```diff
@@ -136,7 +136,9 @@ describe('system test (type 2) - double delegate registrations', () => {
 
 					it('adding to pool delegate registration from same account should fail', done => {
 						localCommon.addTransaction(library, transaction2, err => {
-							expect(err).to.equal('Account is already a delegate');
+							expect(err).to.equal(
+								`Transaction is already confirmed: ${transaction2.id}`
+							);
```

Comment:
The test case here 

```
adding to pool delegate registration from same account should fail
```

Does not mean that we will use same transaction. It means we will use same account. Either update the test description or expectation.